interpreter/cling/lib/MetaProcessor/MetaSema.cpp

```diff
@@ -320,9 +320,13 @@ namespace cling {
       " ==============================================================================\n"
       " Syntax: " << metaString << "Command [arg0 arg1 ... argN]\n"
       "\n"
-      "   " << metaString << "L <filename>\t\t- Load the given file or library\n\n"
-
-      "   " << metaString << "(x|X) <filename>[args]\t- Same as .L and runs a function with"
+      "   " << metaString << "Tab\t\t- Autocomplete or print suggestions\n"
+      "\n"
+      "   " << metaString << "L <filename>[flags]\t\t- Load the given file or library."
+                             "\n\t\t\t\t  Flags: add '+' to compile or '++' to force recompile."
+                             "\n\t\t\t\t  Type .? TSystem::CompileMacro for additional flags.\n"
```

Comment:
This is ROOT-specific (handled in `core/metacling/src/TCling.cxx`).  I think we should move this to `TApplication.cxx`.